api/src/infrastructure/clients/mongodb/MongoDatabaseClient.py

```diff
@@ -22,17 +23,22 @@
 
 
 class MongoDatabaseClient(ClientInterface[Dict, str]):
-    def __init__(self, collection_name: str, client: MongoClient = DEFAULT_MONGO_CLIENT):
-        self.handler = client[config.MONGODB_DATABASE]
-        self.collection: Collection = self.handler[collection_name]
+    def __init__(self, collection_name: str, database_name: str, client: MongoClient = DEFAULT_MONGO_CLIENT):
+        database: Database = client[database_name]
+        self.database = database
+        self.collection_name = collection_name
+        self.collection = database[collection_name]
```

Comment:
Can these values be extracted from the client in an easy way?
Seems unnecessary to pass them when we are also passing the client

Reply:
Nope, these can not be extracted from the client.